.github/workflows/sync-staging.yml

```diff
@@ -56,4 +50,4 @@ jobs:
           git pull
           git checkout staging
           git merge main --no-ff
-          git push origin staging
+          git push origin staging
```

Comment:
this is minor but a more descriptive name here. 
https://github.com/SeattleColleges/nsc-events-nextjs/blob/main/.github/workflows/sync-staging.yml

Reply:
Is this for the name of the job? The link has the same code. Is there any more context for this?

Reply:
I think he's referencing the run-name?

What you have here:
`run-name: Sync-Staging`
What Gary has there:
`run-name: Merge main into Staging`